fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/logging/LogMessageKeys.java

```diff
@@ -65,12 +80,14 @@
     EXPECTED_COLUMN_SIZE("expected_column_size"),
     ACTUAL_COLUMN_SIZE("actual_column_size"),
     KEY_EXPRESSION("key_expression"),
+    KEY_EXPRESSION_C("keyExpression"),
```

Comment:
I guess in line with https://github.com/FoundationDB/fdb-record-layer/pull/611/files#r287116009, I think the `_C` expressions should be converted where used to the under_score version of the key, especially as fixing that later would require removing keys from an enum, which is technically a breaking change.

Reply:
DONE
- Normalized all keys to the underscore variant.
- Does this warrant an update to ReleaseNotes?